src_assets/common/assets/web/header-no-nav.html

```diff
@@ -4,7 +4,31 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="apple-mobile-web-app-capable" content="yes" />
+    <meta name="theme-color" content="#ffc400" />
     <title>Sunshine</title>
+    <link rel="icon" type="image/x-icon" href="/images/favicon.ico">
+    <link rel="apple-touch-icon" href="/images/apple-touch-icon-152x152.png" />
+    <link rel="apple-touch-icon-precomposed" sizes="57x57" href="/images/apple-touch-icon-57x57.png" />
+    <link rel="apple-touch-icon-precomposed" sizes="114x114" href="/images/apple-touch-icon-114x114.png" />
+    <link rel="apple-touch-icon-precomposed" sizes="72x72" href="/images/apple-touch-icon-72x72.png" />
+    <link rel="apple-touch-icon-precomposed" sizes="144x144" href="/images/apple-touch-icon-144x144.png" />
+    <link rel="apple-touch-icon-precomposed" sizes="60x60" href="/images/apple-touch-icon-60x60.png" />
+    <link rel="apple-touch-icon-precomposed" sizes="120x120" href="/images/apple-touch-icon-120x120.png" />
+    <link rel="apple-touch-icon-precomposed" sizes="76x76" href="/images/apple-touch-icon-76x76.png" />
+    <link rel="apple-touch-icon-precomposed" sizes="152x152" href="/images/apple-touch-icon-152x152.png" />
+    <link rel="icon" type="image/png" href="/images/favicon-196x196.png" sizes="196x196" />
+    <link rel="icon" type="image/png" href="/images/favicon-96x96.png" sizes="96x96" />
+    <link rel="icon" type="image/png" href="/images/favicon-32x32.png" sizes="32x32" />
+    <link rel="icon" type="image/png" href="/images/favicon-16x16.png" sizes="16x16" />
+    <link rel="icon" type="image/png" href="/images/favicon-128.png" sizes="128x128" />
```

Comment:
What's the point of all these icon files? The `favicon.ico` already has many sizes embedded inside it.

At least the original one... I'm curious what changed in this file?

Reply:
They may not be all needed, but they were the result of a tool that generates icons for various browsers.

favicon.ico file as well, although maybe that file already had all the sizes for that format.

Reply:
We have these sizes in the existing ico file.

![image](https://user-images.githubusercontent.com/42013603/224369044-d0de3073-2e06-4943-8a67-d833db5fe732.png)

And your modified one removes a bunch of sizes.
![image](https://user-images.githubusercontent.com/42013603/224369631-37432456-d46d-465c-ac45-9e0a6bb11075.png)

Probably best to revert this change.

What tool did you use?

Reply:
Yeah, the previous one was better.

The specific tool I used was http://favicomatic.com

Reply:
It's common to need lots of favicons like this for various devices like ipads, surfaces, android phones, apple phones, etc. The tools like the one linked work really well for this.

Reply:
I've kept the previous favicon.ico just because, but the other images were automatically generated by https://realfavicongenerator.net/